cmd/crio/config.go

```diff
@@ -83,8 +87,8 @@ stream_tls_ca = "{{ .StreamTLSCA }}"
 # "<ulimit name>=<soft limit>:<hard limit>", for example:
 # "nofile=1024:2048"
 # If nothing is set here, settings will be inherited from the CRI-O daemon
-#default_ulimits = [
-{{ range $ulimit := .DefaultUlimits }}{{ printf "#\t%q,\n" $ulimit }}{{ end }}#]
+default_ulimits = [
+{{ range $ulimit := .DefaultUlimits }}{{ printf "\t%q,\n" $ulimit }}{{ end }}]
```

Comment:
I'm assuming it's ok for the last option to have a comma at the end of it and no option afterwards?

Reply:
yes